### PR TITLE
Pinning openssl to fix failing unittest on mac

### PR DESCRIPTION
**Pull request by AliceBalfanz**
[Description of PR]
This PR only adds openssl to the dependencies of xcube, and pins it to the last working version. This should be undone, once https://github.com/conda/conda/issues/11111 is solved. 

No code has changed. 

FYI @forman @TonioF 

Checklist:

* [ ] ~~Add unit tests and/or doctests in docstrings~~
* [ ] ~~Add docstrings and API docs for any new/modified user-facing classes and functions~~
* [ ] ~~New/modified features documented in `docs/source/*`~~
* [ ] ~~Changes documented in `CHANGES.md`~~
* [x] AppVeyor CI passes
* [ ] Test coverage remains or increases (target 100%)

Remember to close associated issues after merge!

### CODE REVIEW

**Comment by AliceBalfanz** [comment]
Docker builds failing due to open PR: https://github.com/dcs4cop/xcube/pull/575

**Review by pont-us — APPROVED** [review]
Looks good!

**Review by dzelge — APPROVED** [review]
We have to check, whether this has implications on the cluster deployments.

**Review by dzelge — CHANGES_REQUESTED** [review]
Sorry, had to revoke my approval. Unittests fail.

**Review by dzelge — APPROVED** [review]

File: .github/workflows/xcube_workflow.yaml [hunk 1]
```diff
@@ -31,7 +31,7 @@ jobs:
         shell: bash -l {0}
         run: |
           pip install pytest pytest-cov
-          mamba install -c conda-forge  flask-testing moto requests-mock
+          mamba install -c conda-forge  flask-testing requests-mock
```
Comment (dzelge) [inline .github/workflows/xcube_workflow.yaml:34]:
```suggestion
```

**Review by dzelge — CHANGES_REQUESTED** [review]
I would remove the whole line.I don't think that flask-testing and requests-mock is actually beebn used in xcube ubnittests

**Pull request merged**